rkt/api_service.go

```diff
@@ -562,16 +562,24 @@ func fillPodDetails(store *imagestore.Store, p *pod, v1pod *v1alpha.Pod) {
 		if err != nil {
 			stderr.PrintE(fmt.Sprintf("failed to get the container PID1 for pod %q", p.uuid), err)
 		} else {
-			cgroup, err := cgroup.GetCgroupPathByPid(pid, "name=systemd")
+			isUnified, err := cgroup.IsCgroupUnified("/")
+			if err != nil {
+				stderr.PrintE("failed to determine the cgroup version", err)
```

Comment:
But that's `IsIsolatorSupported()`, not `IsCgroupUnified()`. Am I missing something?